src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -495,8 +495,36 @@ def get_observation(self, event: Event) -> pd.DataFrame:
         return pd.concat([df_creation, df_death]).sort_values(["event_date", "date_of_birth"])
 
 
+class TaxObserver:
+    """Holder for three interdependent observers relevant to tax data"""
+
+    @property
+    def name(self):
+        return f"tax_observer"
+
+    def __init__(self):
+        tax_w2 = TaxW2Observer()
+        tax_1040 = Tax1040Observer(tax_w2)
+        tax_dependents = TaxDependentsObserver(tax_w2)
+
+        self._sub_components = [
+            tax_w2,
+            tax_1040,
+            tax_dependents,  # NOTE: this component must come after the TaxW2Observer to function correctly
```

Comment:
We should remove these notes now, I think?

Reply:
good call